### ISSUE TITLE
Publish on f-droid

### ISSUE THREAD

**Issue by muelli** [kaputnikGo/PilferShush_prod#1]
It'd be amazing if this package was available on F-Droid.

**Comment by kaputnikGo** [comment]
yeah maybe one day soon and maybe with just the nuhf audio scanner components. currently there is PilferShush Jammer on fdroid which has nuhf jamming and user app scans, nuhf beacon sdk scans.

**Comment by IzzySoft** [comment]
Publishing the app in the official repo would also require a license to be specified. I couldn't figure which one this app is using – maybe you want to add this, @kaputnikGo? Am I guessing correctly that it uses Apache-2.0 like PilferShush?

I've meanwhile added your app to [my repository](https://apt.izzysoft.de/fdroid/repo) (hope this is OK with you; once it gets added to the official repo, I'll remove it in mine again).

**Comment by IzzySoft** [comment]
I see. If you want me to update the description (see [here](http://apt.izzysoft.de/fdroid/index/apk/cityfreqs.com.pilfershush_prod)) just let me know, and I'll of course do so! If you change the app name, that should be reflected automatically.

**Comment by kaputnikGo** [comment]
This app is being updated to run as an audio scanner only (specifically Near Ultra-High Frequency). The jammer app is available on GPlay, FDroid and source at https://github.com/kaputnikGo/PilferShushJammer

Reasoning is I need a dedicated scanner for a privacy project coming up soon and the Jammer app is already taking up a lot of time just adapting to Android API changes to remain a viable microphone jamming app.

**Comment by IzzySoft** [comment]
Thanks! Updated (together with the Github URLs). Will take effect with the next sync this evening – when the update will be pushed as well.
